src/ppx/application.cpp

```diff
@@ -998,6 +998,21 @@ void Application::DispatchConfig()
         mSettings.window.title = mSettings.appName;
     }
 
+    // ImGui doesn't support prebuilt command buffer.
+    if (mSettings.grfx.enablePreRecordCmd) {
+        PPX_LOG_WARN("Disable ImGui when enable prebuilt command buffer.");
```

Comment:
Can you reword the warning message to be similar to others, e.g. "ImGui does not support prebuilt command buffers, will be disabled"